crazyflie/config/crazyflies.yaml

```diff
@@ -1,14 +1,64 @@
-cf231:
-  initial_position:
-  - 0
-  - 0
-  - 0
-  type: cf21
-  uri: radio://0/80/2M/E7E7E7E7E7
-cf5:
-  initial_position:
-  - 0
-  - -0.5
-  - 0
-  type: cf21
-  uri: radio://0/80/2M/E7E7E7E705
+# named list of all robots
+robots:
+  cf231:
+    enabled: true
+    uri: radio://0/80/2M/E7E7E7E7E7
+    initial_position: [0, 0, 0]
+    type: cf21
+    # firmware_params:
+    #   kalman:
+    #     pNAcc_xy: 1.0 # default 0.5
+  cf5:
+    enabled: false
+    uri: radio://0/80/2M/E7E7E7E705
+    initial_position: [0, -0.5, 0]
+    type: cf21
+    # firmware_params:
+    #   kalman:
+    #     pNAcc_xy: 1.0 # default 0.5
+
+# Definition of the various robot types
+robot_types:
+  cf21:
+    motion_capture:
+      enabled: true
+      # only if enabled; see motion_capture.yaml
+      marker: default_single_marker
+      dynamics: default
+    big_quad: false
+    battery:
+      voltage_warning: 3.8  # V
+      voltage_critical: 3.7 # V
```

Comment:
Perhaps an example of firmware parameters here as well, albeit commented out like with the crazyflies?

Reply:
Done!